app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -85,9 +85,10 @@ def uuid_provider_types
       def fetch_server_entities
         @data[:middleware_deployments] = []
         @data[:middleware_datasources] = []
+        @data[:middleware_jms] = []
         @eaps.map do |eap|
-          @ems.child_resources(eap.path).map do |child|
-            next unless child.type_path.end_with?('Deployment', 'Datasource')
+          @ems.child_resources(eap.path, true).map do |child|
+            next unless child.type_path.end_with?('Deployment', 'Datasource', 'JMS%20Topic', 'JMS%20Queue')
```

Comment:
why is the need for `%20` - which component has the limitation that requires it?